src/Microsoft.AspNet.Mvc.ModelBinding/Binders/MutableObjectModelBinder.cs

```diff
@@ -334,29 +337,6 @@ internal static PropertyValidationInfo GetPropertyValidationInfo(ModelBindingCon
                 if (requiredValidator != null)
```

Comment:
Yes. I logged a separate bug for that item. https://github.com/aspnet/Mvc/issues/2407 at this point it's getting hard to track what the impact of #1712 is supposed to be, so it felt like another issue was warranted.